thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionPipelineTaskRunner.java

```diff
@@ -167,6 +175,14 @@ public DetectionPipelineTaskRunner(DetectionConfigManager detectionDAO, MergedAn
       }
       this.detectionDAO.update(config);
 
+      // re-notify the anomalies if any
+      for (MergedAnomalyResultDTO anomaly : result.getAnomalies()) {
+        // if an anomaly should be re-notified, update the notification lookup table in the database
+        if (anomaly.shouldRenotify()) {
```

Comment:
Do we expect that new anomalies also have `renotify` flag as true? Or this should only apply for existing anomalies?

Reply:
no. the renotify flag only applies to exist anomalies.